### PR TITLE
[#948] Eksport af invitations informationer

### PR DESCRIPTION
**Pull request by mhewel**

### CODE REVIEW

**Comment by rasmusselsmark** [comment]
Har testet lokalt, nyt felt i dropdown kom frem under Admin->Invitationer:
image

File: members/admin/activityinvite_admin.py [hunk 1]
```diff
@@ -260,3 +265,68 @@ def participating(self, item):
...
+            "Deltager-email"; "Familie-email"; "Pris"; "Pris note"; "Ekstra email info" ;\
+            "Deltager i aktiviteten"; "Invitationsdato"; "Udløbsdato"; "Afslåetdato"\n"""
+
+        for invitation in queryset:
+            if invitation.person.activityparticipant_set.filter(
+                activity=invitation.activity
+            ).exists():
+                participate = "Ja"
+            else:
+                participate = "Nej"
```
Comment (lakridserne) [inline members/admin/activityinvite_admin.py:280]:
```
participate = "Ja" if invitation.person.activityparticipant_set.filter(
    activity=invitation.activity
).exists() else "Nej"
```

File: members/admin/activityinvite_admin.py [hunk 2]
```diff
@@ -260,3 +265,68 @@ def participating(self, item):
...
+            if invitation.expire_dtm is None:
+                expire_date = ""
+            else:
+                expire_date = invitation.expire_dtm.strftime("%Y-%m-%d")
+            if invitation.rejected_at is None:
+                rejected_date = ""
+            else:
+                rejected_date = invitation.rejected_at.strftime("%Y-%m-%d")
```
Comment (lakridserne) [inline members/admin/activityinvite_admin.py:288]: Brug ternary operator

**Review by lakridserne — CHANGES_REQUESTED** [review]
Ser fint ud bortset fra at vi bør bruge ternary operator.

File: members/admin/activityinvite_admin.py (hunk 2 above)
Comment (rasmusselsmark) [reply members/admin/activityinvite_admin.py]: Done

File: members/admin/activityinvite_admin.py (hunk 1 above)
Comment (rasmusselsmark) [reply members/admin/activityinvite_admin.py]: Done

**Review by lakridserne — APPROVED** [review]

**Review by lakridserne — APPROVED** [review]

**Pull request merged**